commcare_connect/templates/tailwind/base.html

```diff
@@ -26,6 +27,25 @@
       {% include 'tailwind/layouts/header.html'%}
       <div class="h-5 w-full mt-16"></div>
       <main id="content" class="md:max-w-screen-2xl mx-auto">
+        {% if messages %}
```

Comment:
did we always need to manually render messages? or is that new?

Reply:
Yeah, it has been manual.